frontend/src/App.js

```diff
@@ -19,7 +29,70 @@ const App = () => {
           <Footer/>
         </Router>
       </div>
+
+
+      <h1 className="our_impact" ><b>OUR IMPACT</b></h1>
```

Comment:
Hey @SharmaTanish , Please make a separate component for that, for reference you can see `frontend/src/Components/Navbar/`.

App.js Use only for a component call with the router.